src/solve/opaque-types.md

```diff
@@ -99,8 +104,11 @@ The handling of member constraints does not change in the new solver. See the
 
 ## calling methods on opaque types
 
+<!--
 FIXME: We need to continue to support calling methods on still unconstrained
 opaque types in their defining scope. It's unclear how to best do this.
+-->
```

Comment:
I think a FIXME is nice here since otherwise it seems like this header was just intentionally left blank with little explanation